server/src/services/ito/itoService.ts

```diff
@@ -136,10 +142,63 @@ function dbToAdvancedSettingsPb(
     updatedAt: dbAdvancedSettings.updated_at.toISOString(),
     llm: create(LlmSettingsSchema, {
       asrModel: dbAdvancedSettings.llm.asr_model,
+      asrPrompt: dbAdvancedSettings.llm.asr_prompt,
+      asrProvider: dbAdvancedSettings.llm.asr_provider,
+      llmProvider: dbAdvancedSettings.llm.llm_provider,
+      llmTemperature: dbAdvancedSettings.llm.llm_temperature,
+      llmModel: dbAdvancedSettings.llm.llm_model,
+      transcriptionPrompt: dbAdvancedSettings.llm.transcription_prompt,
+      editingPrompt: dbAdvancedSettings.llm.editing_prompt,
+      noSpeechThreshold: dbAdvancedSettings.llm.no_speech_threshold,
+      lowQualityThreshold: dbAdvancedSettings.llm.low_quality_threshold,
     }),
   })
 }
 
+async function transcribeWithGroq(
```

Comment:
We should drop the `withGroq` suffix as we'll be adding more providers, and the asr provider is one of the advanced settings. You don't need to add a switch statement baed on that asr provider as part of this pr -- I'll do it in mine where I'm adding cerberas